main/api/v1/auth.py

```diff
@@ -16,8 +18,13 @@
 @api_v1.resource('/auth/signin/', endpoint='api.auth.signin')
 class AuthAPI(restful.Resource):
   def post(self):
-    username = util.param('username') or util.param('email')
-    password = util.param('password')
+    args = parser.parse({
+        'username': wf.Str(missing=None),
+        'email': wf.Str(missing=None),
+        'password': wf.Str(missing=None),
+      })
```

Comment:
I'm just curious on why not using the `util.param('..')` instead here.. since that is used as well?

Just to have a better control of what arguments are expected or?!